fvm/evm/stdlib/contract.go

```diff
@@ -209,15 +135,6 @@ func newInternalEVMTypeEncodeABIFunction(
 				panic(errors.NewUnreachableError())
 			}
 
```

Comment:
how come this is removed now?

Reply:
fixed it, it was somehow not synced with the latest version of master.